### DISCUSSION TITLE
点击错误卡片引发刷新

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [Meloong-Git/PCL#4501] [讨论]
不确认是否是 Bug，先扔在这里
PCL（tangge233's Fork）CI Build 231，Debug Verison

[Video](https://github.com/user-attachments/assets/985163d4-ff4b-4da9-aeb0-7b666e757bcb)
[Log1.txt](https://github.com/user-attachments/files/16621788/Log1.txt)

**Comment by LTCatt** [comment]
这真是特性
